core/src/main/scala/kafka/log/Log.scala

```diff
@@ -68,8 +67,12 @@ case class LogAppendInfo(var firstOffset: Long,
                          targetCodec: CompressionCodec,
                          shallowCount: Int,
                          validBytes: Int,
-                         offsetsMonotonic: Boolean)
-
+                         offsetsMonotonic: Boolean,
+                         pid: Long,
+                         epoch: Short,
+                         firstSequence: Int,
+                         lastSequence: Int,
+                         consecutiveSequenceNumbers: Boolean)
```

Comment:
What is `consecutiveSequenceNumbers` supposed to mean? Do we plan on allowing non-continous sequnece numbers?